### ISSUE TITLE
[Feature Request] (LINUX) Change the icon as it use the default X icon

### ISSUE THREAD

**Issue by cylian914** [FTBTeam/FTB-App#838]
### What is your suggestion?

On linux (at least archlinux) the app don't have a icon, it use the default X11 icon
![image](https://user-images.githubusercontent.com/73445993/212569744-b9a550e4-b600-44b4-a573-ce6dc6cb508c.png)
i can be easly change by the favicon of the website (https://www.feed-the-beast.com/fav/favicon-32x32.png)

**Comment by lupusagax** [comment]
I can confirm the same behavior. Kubuntu 22.04, KDE Plasma 5.24.7.

Icon shows on desktop config file that is added, but does not show on taskbar.

**Comment by Declan-F** [comment]
I believe I have this working, though it's a bit of a hack. The first thing to note is that there are two icons, the development one (handled directly by electron), and the release one (handled by electron builder). Neither work at the moment, but the one that I would consider of importance is the release one. 

From what I have found, the **dir** distribution release does not seem to play well with icons. Windows doesn't use it, though mac does. I _think_ it might not have the favicon functionality implemented for it on linux, but I cannot say for certain. In the end I swapped it to flatpak.

After messing with it more, the other issue was that the program thought the image was a zero by zero png. This is seemingly somehow a result of [this](https://github.com/develar/app-builder/blob/1cceb8fd6fe68955cecb96fb48259087980cd826/pkg/icons/icon-converter.go#L187) line of code which I found through https://github.com/develar/app-builder/issues/42. The if statement above that line means changing the file ending to an icns file would work, though flatpak does not support 1028x1028 image files, so I had to create an extra image other than the already present icns. 

Here's an image of the working setup. I'll work on submitting a pull request.
![Screenshot from 2024-02-21 18-04-44](https://github.com/FTBTeam/FTB-App/assets/92695239/beff48c9-c138-41f6-9af1-f1cd9aecc9b8)
